### PR TITLE
Remove ynh_detect_arch helper

### PR DESCRIPTION
**Pull request by ericgaspar**
`ynh_detect_arch` is deprecated in favor of `YNH_ARCH`

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -31,7 +31,7 @@ email=$(ynh_user_get_info --username=$admin --key=mail)
 password=$YNH_APP_ARG_PASSWORD
 language=$YNH_APP_ARG_LANGUAGE
 team_display_name=$YNH_APP_ARG_TEAM_DISPLAY_NAME
-architecture=$(ynh_detect_arch)
+architecture=$YNH_ARCH
```
Comment (kemenaran) [inline scripts/install:34]: This requires Yunohost >= 4.3, right? Should the minimum version in the manifest be updated as well?

**Review by kemenaran — CHANGES_REQUESTED** [review]
This

**Comment by ericgaspar** [comment]
!testme

**Review by kemenaran — APPROVED** [review]

**Comment by ericgaspar** [comment]
!tastemetitus

**Pull request merged**